packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -58,6 +58,16 @@ export class HdPaths {
     return HdPaths.bip44Like(HdPaths.coinTypes.iov, account);
   }
 
+  /**
+   * A second generation IOV HD path in the form m/44'/234'/0'/0/a.
+   * This is used on the Cosmos SDK based chain starting from summer 2020.
+   *
+   * @param account The account index `a` starting at 0
+   */
+  public static iovSecondGen(account: number): readonly Slip10RawIndex[] {
```

Comment:
Let's be more explicit with the name. Any protest against `iovCosmosSdk`?

Reply:
No protest, will update

Reply:
updated